6.0/6.0.1/04-advanced_configuration.md

```diff
@@ -0,0 +1,158 @@
+<!-- Space: TOOLS -->
+<!-- Parent: Installer -->
+<!-- Title: Advanced Configuration -->
+<!-- Layout: plain -->
+
+# Advanced configuration
+
+<br />
+
+<!-- Include: ac:toc -->
+
+<br />
+
+## Use hostPath for Static Storage of Sysdig Components
+
+As described in the Installation Storage Requirements, the Installer assumes usage of a dynamic storage provider (AWS or GKE). In case these are not used in your environment, add the entries below to the values.yaml to configure static storage.
```

Comment:
Please hyperlink to Installation Storage Requirements

Reply:
Could you share the right link @radhikapc? not sure if @bennythejudge could help here?

Reply:
I do not know which section this refers to

Reply:
In the current documentation, we have the same paragraph and it is not pointing to any link, could you check here @radhikapc:
https://docs.sysdig.com/en/docs/administration/on-premises-deployments/on-premises-installation/installer-kubernetes-openshift/frequently-used-installer-configurations/#use-hostpath-for-static-storage-of-sysdig-components

Reply:
@bennythejudge Can we update this. We can add an exampleHostpath values.yaml and point folks from here to that example. The current example we have under `6.0/6.0.1/examples/openshift-with-hostpath/values.yaml` is insufficient wdyt? 

It does not cover Kafka for one. It also does not cover the sizing spec.

Reply:
@pravin-jothinathan I have submitted https://github.com/draios/installer/pull/2562 with various improvements including for `hostPath`.